### ISSUE TITLE
On the problem that SCT is larger than GT

### ISSUE THREAD

**Issue by haotangyuan** [AustinTapp/CM3dLDM#1]
Dear author, 
I am a beginner in image generation.
Why do I have the problem that SCT is larger than GT in the reasoning process after training the syn2023 dataset, and the shape size of the image is consistent? I am sure I have not changed infer.py.
In this picture, Mr is on the left, noise and Z are in the middle, and GT is on the right
What should I do to get the correct generated results？
Looking forward to your reply. Thank you very much!!!

![Image](https://github.com/user-attachments/assets/fd283861-d325-486a-9668-47be368c4a93)

**Comment by AustinTapp** [comment]
Hi @haotangyuan,

Could you please share your input and generated .nii.gz images through a Google Drive or similar link? I need to check if you are asking about the image being physically larger in voxel dimensions or just appearing larger (i.e., contains less background compared to the GT).

If it is the latter, this might be due to incomplete training convergence or latent space misalignment, which can cause the model to over-represent certain structures. In latent diffusion models, the latent space captures compressed structural features. Ensuring the model properly learns anatomical consistency will help the generated sCT match the GT in a perceptually similar manner. By increasing the number of training epochs, adjusting learning rates, or refining your loss weights, you might improve how the model learns the mapping from MR to sCT.

Best,
Austin
